home.css

```diff
@@ -0,0 +1,226 @@
+body{
+    
+    margin: 0%;
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+}
+
+.header
+{
+    background-color: black;
+    margin: 0px;
+}
+
+.nav {
+    display: flex;
+    height: 80px;
+    
+    font-size: large;
+    border-bottom-style: dotted;
+    border-color: white;
+    border-radius: 35px;
+    
+}
+
+
+.imge {
+    width: 40px;
+    height: 40px;
+   margin-top: 1%;
+    margin-left: 4%
+}
+
+.container {
+    list-style-type: none;
+    /* Remove bullets */
+    padding: 0;
+    /* Remove padding */
+    /* Remove margins */
+    display: flex; 
+    margin-left: 30%;
+}
+
+.log{
+    color: white;  
+    background-color: red;
+    border-radius: 5px;
+    margin-left: 25%;
+    margin-top: 1%;
+    width: 70px;
+    height: 40px;
+    font-weight: bolder;
+}
+
+
+.conatiner1{
+    display: flex;
+    width: 65%;
+    margin: auto;
+    margin-top: 10px;
+    height: 300px;
+    background-color: rgb(7, 7, 7);
+    border-radius: 10px;
+}
+
+
+
+.secbutton{
+    background-color: red;
+    color: white;
+    font-weight: bolder;
+    font-size: 85%;
+   
+    margin-top: 3%;
+    width: 100px;
+    height: 50px;
+}
+
+.sideimage{
```

Comment:
Please give css class and ids names in kebab case. Like "image-container".

Please make changes at all places in css files